web/modules/custom/collection/collection.behat.inc

```diff
@@ -157,6 +167,36 @@ class CollectionSubContext extends DrupalSubContextBase implements DrupalSubCont
   }
 
   /**
+   * Indexes items of entity type: collection.
+   *
+   * In addiction, views cache data are being cleared.
+   *
+   * @throws \Exception
+   *   When no items are indexed.
+   *
+   * @Given The collections are indexed
+   */
+  public function theCollectionsAreIndexed() {
+    /** @var String $id */
+    $id = 'collections';
+
+    /** @var String $views_id */
+    $views_id = 'collection';
+
+    /** @var \Drupal\search_api\IndexInterface $index */
+    $index = \Drupal\search_api\Entity\Index::load($id);
+    if (empty($index)) {
+      throw new \Exception("Collection index not found.");
+    }
+    $index->reindex();
+    $index->indexItems();
+
+    // Clear the views cache data.
+    $view_tags = \Drupal\views\Views::getView($views_id)->getCacheTags();
+    \Drupal\Core\Cache\Cache::invalidateTags($view_tags);
```

Comment:
I don't understand this. Why do we manually need to clear the Views cache when new items are indexed? This should be taken care of automatically.

So in real life when items are indexed they do not appear in Views until you manually clear the cache? That seems like a bug to me.